libs/fabric/src/lib/components/marquee-selection/marquee-selection.component.ts

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import {
+  ChangeDetectionStrategy,
+  ChangeDetectorRef,
+  Component,
+  ElementRef,
+  Input,
+  NgZone,
+  Renderer2,
+  ViewChild,
+} from '@angular/core';
+import { ReactWrapperComponent } from '@angular-react/core';
+import { IMarqueeSelectionProps } from 'office-ui-fabric-react/lib/MarqueeSelection';
+
+@Component({
+  selector: 'fab-marquee-selection',
+  exportAs: 'fabMarqueeSelection',
+  template: `
+    <MarqueeSelection
+      #reactNode
+      [selection]="selection"
+      [className]="className"
+      [componentRef]="componentRef"
+      [isDraggingConstrainedToRoot]="isDraggingConstrainedToRoot"
+      [isEnabled]="isEnabled"
+      [rootProps]="rootProps"
+      [styles]="styles"
+      [theme]="theme"
+      [ShouldStartSelection]="onShouldStartSelection">
+      <ReactContent><ng-content></ng-content></ReactContent>
+    </MarqueeSelection>
+  `,
+  styles: ['react-renderer'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class FabMarqueeSelectionComponent extends ReactWrapperComponent<IMarqueeSelectionProps> {
+  
+  @ViewChild('reactNode')
+  protected reactNodeRef: ElementRef;
+
+  // Required members
+  @Input()
+  selection: IMarqueeSelectionProps['selection'];
+
+  // Optional members
```

Comment:
This was for my own organization since I was going off the list of properties in the latest online Fabric documentation. I can change it to match the way we're doing it in other files